packages/devtools-local-toolbox/public/js/index.js

```diff
@@ -47,6 +45,16 @@ function initApp() {
 function renderRoot(_React, _ReactDOM, component, _store) {
   const mount = document.querySelector("#mount");
 
+  // Using this static variable allows webpack to know at compile-time
+  // to avoid this require and not include it at all in the output.
+  if (process.env.TARGET !== "firefox-panel") {
+    const theme = getValue("theme");
+    theme == "dark" ? require("./lib/themes/dark-theme.css")
+                    : require("./lib/themes/light-theme.css");
+
+    document.body.parentNode.classList.add(`theme-${theme}`);
```

Comment:
The dark theme requires adding a class to the `<html>` element. This patch adds the class to both the html element and the root, which is just a way of being consistent in both cases